GTG/gtk/browser/browser.py

```diff
@@ -656,13 +640,14 @@ def on_workview_toggled(self, widget):
         self.do_toggle_workview()
 
     def on_sidebar_toggled(self, widget):
-        view_sidebar = self.builder.get_object("view_sidebar")
+        pass
+        #view_sidebar = self.builder.get_object("view_sidebar")
```

Comment:
can you repurpose the code to work with your new `tags` button? uncomment it and change view_sidebar to tags